taccsite_cms/_settings/form_plugin.py

```diff
@@ -67,26 +75,24 @@
     (
         _('Submission settings'),
         {
-            'description': 'Whether to save form data.',
+            'description': 'You can choose to set from which, and to which, e-mail address to send form submissions and/or to save submitted form data in the CMS database. <small>The "SENDER_EMAIL_ADDRESS" may remain blank, so server default no-reply address would be used.</small>',
```

Comment:
This is the default text, with `<small>` text I appended to explain why one may want `email_from` not set.

Reply:
Could maybe be worded as
```
<small>Portal default from address (DEFAULT_FROM_EMAIL) will be used if "SENDER_EMAIL_ADDRESS" is blank.</small>
```

Reply:
Yes, better.

Okay. And I'll tweak it to not assume a portal, and fix my original typo, and add some formatting.

```suggestion
            'description': 'You can choose to set from which, and to which, e-mail address to send form submissions and/or to save submitted form data in the CMS database. <small>The CMS default <em>from</em> address (<code>DEFAULT_FROM_EMAIL</code>) will be used if "SENDER EMAIL ADDRESS" is blank.</small>',
```